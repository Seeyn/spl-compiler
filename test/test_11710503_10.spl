int nest(int i){
    return i;
}
int main(){
    int a = 0;
    float b;
    a = nest(nest(a));
    b =  nest(nest(a));
}