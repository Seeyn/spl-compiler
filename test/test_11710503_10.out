Error type 5 at Line 8: unmatching type on both sides of assignment
